bot.rb

```diff
@@ -434,6 +458,23 @@ def fix_checkboxes(text)
 
       return new_text if new_text != text
     end
+
+    # Checks if a PR's description contains an issue reference.
+    def referenced_issue_number?(pr)
+      return unless pr.body
+
+      # Searching for `closes #1234` or `fixes #1234` in PR's description
+      issue_number = pr.body[/(#{ISSUE_CLOSING_KEYWORDS.join('|')}) #\d{1,}/i, 0]
+      issue_number = issue_number[/#\d{1,}/i, 0] if issue_number
+      issue_number = issue_number.tr('#', '') if issue_number
+
+      # Searching for `closes https://github.com/REPOSITORY_OWNER/REPOSITORY_NAME/issues/1234` 
+      # or `fixes https://github.com/fastlane/REPOSITORY_OWNER/REPOSITORY_NAME/1234 in PR's description
```

Comment:
Oh, I believe we can update the description of this comment: 

```
# Searching for issue closing keywords + issue URL in PR's description, i.e. `closes https://github.com/REPOSITORY_OWNER/REPOSITORY_NAME/issues/1234` 
```

WDYT ? 🤔

Reply:
Yep.